internal/service/ipam/api_server_resource_test.go

```diff
@@ -0,0 +1,1089 @@
+	}]
+	depends_on = [bloxone_dns_auth_zone.test_zone]
+}
+`, zoneFQDN, name)
+}
+
+func testAccServerDhcpConfig(name string, allowUnknown, allowUnknownV6, ignoreClientUid bool,
+	leaseTime, leaseTimeV6 int) string {
+	return fmt.Sprintf(`
+resource "bloxone_dhcp_server" "test_dhcp_config" {
+    name = %q
+    dhcp_config = {
+		allow_unknown = %t
+		allow_unknown_v6 = %t
+		ignore_client_uid = %t
+		lease_time = %d
+		lease_time_v6 = %d
+	}
+}
+`, name, allowUnknown, allowUnknownV6, ignoreClientUid, leaseTime, leaseTimeV6)
+}
+
+func testAccServerGssTsigFallback(name string, gssTsigFallback string) string {
+	return fmt.Sprintf(`
+resource "bloxone_dhcp_server" "test_gss_tsig_fallback" {
+    name = %q
+    gss_tsig_fallback = %q
+}
+`, name, gssTsigFallback)
+}
+
+func testAccServerHeaderOptionFilename(name string, headerOptionFilename string) string {
+	return fmt.Sprintf(`
+resource "bloxone_dhcp_server" "test_header_option_filename" {
+    name = %q
+    header_option_filename = %q
+}
+`, name, headerOptionFilename)
+}
+
+func testAccServerHeaderOptionServerAddress(name string, headerOptionServerAddress string) string {
+	return fmt.Sprintf(`
+resource "bloxone_dhcp_server" "test_header_option_server_address" {
+    name = %q
+    header_option_server_address = %q
+}
+`, name, headerOptionServerAddress)
+}
+
+func testAccServerHeaderOptionServerName(name string, headerOptionServerName string) string {
+	return fmt.Sprintf(`
+resource "bloxone_dhcp_server" "test_header_option_server_name" {
+    name = %q
+    header_option_server_name = %q
+}
+`, name, headerOptionServerName)
+}
+
+func testAccServerHostnameRewriteChar(name string, hostnameRewriteChar string) string {
+	return fmt.Sprintf(`
+resource "bloxone_dhcp_server" "test_hostname_rewrite_char" {
+    name = %q
+    hostname_rewrite_char = %q
+}
+`, name, hostnameRewriteChar)
+}
+
+func testAccServerHostnameRewriteEnabled(name string, hostnameRewriteEnabled string) string {
+	return fmt.Sprintf(`
+resource "bloxone_dhcp_server" "test_hostname_rewrite_enabled" {
+    name = %q
+    hostname_rewrite_enabled = %q
+}
+`, name, hostnameRewriteEnabled)
+}
+
+func testAccServerHostnameRewriteRegex(name string, hostnameRewriteRegex string) string {
+	return fmt.Sprintf(`
+resource "bloxone_dhcp_server" "test_hostname_rewrite_regex" {
+    name = %q
+    hostname_rewrite_regex = %q
+}
+`, name, hostnameRewriteRegex)
+}
+
+func testAccServerName(name string) string {
+	return fmt.Sprintf(`
+resource "bloxone_dhcp_server" "test_name" {
+    name = %q
+}
+`, name)
+}
+
+func testAccServerServerPrincipal(name string, serverPrincipal string) string {
+	return fmt.Sprintf(`
+resource "bloxone_dhcp_server" "test_server_principal" {
+    name = %q
+    server_principal = %q
+}
+`, name, serverPrincipal)
+}
+
+func testAccServerTags(name string, tags map[string]string) string {
+	tagsStr := "{\n"
+	for k, v := range tags {
+		tagsStr += fmt.Sprintf(`
+		%s = %q
+`, k, v)
+	}
```

Comment:
```suggestion
		tagsStr += fmt.Sprintf(`%s = %q`, k, v)
	}
```

Reply:
Test case fails with this error when I add the above suggested   "Expected a newline or comma to mark the beginning of the next attribute"